devito/passes/iet/definitions.py

```diff
@@ -260,17 +261,24 @@ def place_definitions(self, iet, **kwargs):
 
         return iet, {}
 
+    @iet_visit
+    def derive_transfers(self, iet):
+        """
+        Collect all symbols that cause host-device data transfer, distinguishing
+        between reads and writes.
+        """
+        return ([], [])
+
     @iet_pass
-    def map_onmemspace(self, iet, **kwargs):
+    def place_transfers(self, iet, **kwargs):
```

Comment:
map_transfers?

Reply:
no, it's more of a placement than a mapping. "map" is the process of binding (ie mapping) two things. Nitpicking, anyway ;)